### PR TITLE
different layout options for screen?

### PR DESCRIPTION
**Pull request by karthikv**
are we going to have different layout options? currently there is one where header has 3 columns, body has 2 columns and footer has 3 columns.

### CODE REVIEW

**Comment by karthikv** [comment]
@ednapiranha @fwenzel Just finished adding the layout editor! It's ready for review.

File: lib/screens.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ function getDefaultScreen(req) {
   return {
     title: req.body.title,
     isStart: req.body.isStart || false,
-    layout: req.body.layout,
+    layout: req.body.layout ||
+      [ [ 8, 4 ], [ 4, 4, 4 ], [ 4, 8 ] ],
```
Comment (operatorjen) [inline lib/screens.js]: hmm would prefer if this was set above in a var, with each part on a new line, then that var added in this return

**Comment by operatorjen** [comment]
fix nit and you can merge!

**Pull request merged**